journal-of-music-technology-and-education.csl

```diff
@@ -0,0 +1,585 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Journal of Music Technology and Education</title>
+    <title-short>JMTE</title-short>
+    <id>http://www.zotero.org/styles/journal-of-music-technology-and-education</id>
+    <link href="http://www.zotero.org/styles/journal-of-music-technology-and-education" rel="self"/>
+    <link href="http://owl.english.purdue.edu/owl/resource/560/01/" rel="documentation"/>
```

Comment:
could you replace the link here with a link to the instructions for authors for JMTE, please?